include/dlaf/auxiliary/norm/mc.h

```diff
@@ -78,32 +108,27 @@ dlaf::BaseType<T> Norm<Backend::MC, Device::CPU, T>::max_L(comm::CommunicatorGri
 
   // then it is necessary to reduce max values from all ranks into a single max value for the matrix
 
-  auto max_element = [](std::vector<NormT>&& values) {
-    DLAF_ASSERT(!values.empty(), "");
-    return *std::max_element(values.begin(), values.end());
-  };
-  NormT local_max_value =
-      tiles_max.empty() ? NormT{0}
-                        : sync_wait(ex::when_all_vector(std::move(tiles_max)) |
-                                    dlaf::internal::transform(dlaf::internal::Policy<Backend::MC>(),
-                                                              std::move(max_element)));
-  NormT max_value;
-  dlaf::comm::sync::reduce(comm_grid.rankFullCommunicator(rank), comm_grid.fullCommunicator(), MPI_MAX,
-                           make_data(&local_max_value, 1), make_data(&max_value, 1));
-
-  return max_value;
+  ex::unique_any_sender<NormT> local_max_value = ex::just(NormT{0});
```

Comment:
```suggestion
  ex::unique_any_sender<NormT> local_max_value = ex::just(NormT{});
```
? @rasolca I don't remember what you ended up with some other PR. If I remember correctly you had some corner cases about `T{0}` not doing what you expected?

Reply:
problem was complex in gpu kernel code. No problems on CPU and in this case NormT is always real...